<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerDashboard</class>
 <widget class="QWidget" name="managerDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager Dashboard</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QFrame" name="navigationPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="inventoryButton">
        <property name="text">
         <string>Inventory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="employeeButton">
        <property name="text">
         <string>Employee</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="historyButton">
        <property name="text">
         <string>History</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="serviceReportButton">
        <property name="text">
         <string>Service Report</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="purchaseButton">
        <property name="text">
         <string>Purchase</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logOutButton">
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="inventoryPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="inventoryNavigation">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="addItemButton">
            <property name="text">
             <string>Add Item</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="searchBar">
            <property name="placeholderText">
             <string>Product Name...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="searchButton">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QLabel" name="title">
            <property name="text">
             <string>INVENTORY</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="inventoryList">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="inventoryAttributes">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="productId">
               <property name="text">
                <string>Product ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="productName">
               <property name="text">
                <string>Product Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="quantity">
               <property name="text">
                <string>Quantity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="price">
               <property name="text">
                <string>Price</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="category">
               <property name="text">
                <string>Category</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QColumnView" name="products"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="employeePage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="employeeNavigation">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="addEmployeeButton">
            <property name="text">
             <string>Add Employee</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="searchBar_2">
            <property name="placeholderText">
             <string>Name...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="searchButton_2">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QLabel" name="title_2">
            <property name="text">
             <string>EMPLOYEE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="employeeList">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QFrame" name="employeeAttributes">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="employeeId">
               <property name="text">
                <string>Employee ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="employeeName">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="employeeDateHired">
               <property name="text">
                <string>Date Hired</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="employeeAddress">
               <property name="text">
                <string>Address</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="employeeContact">
               <property name="text">
                <string>Contact Number</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="employeeCharge">
               <property name="text">
                <string>Charge</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QColumnView" name="employees"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="historyPage">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QFrame" name="historyNavigation">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="title_3">
            <property name="text">
             <string>TRANSACTION HISTORY</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="historyList">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QFrame" name="historyAttributes">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="date">
               <property name="text">
                <string>Date</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="customerName">
               <property name="text">
                <string>Customer Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="contactNumber">
               <property name="text">
                <string>Contact Number</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="technician">
               <property name="text">
                <string>Technician</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labchargeel_6">
               <property name="text">
                <string>Charge</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="serviceType">
               <property name="text">
                <string>Service Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="status">
               <property name="text">
                <string>Status</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QColumnView" name="history"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
